incubator/cassandra/templates/statefulset.yaml

```diff
@@ -22,9 +26,6 @@ spec:
 {{ toYaml .Values.podLabels | indent 8 }}
 {{- end }}
     spec:
-{{- if .Values.selector }}
-{{ toYaml .Values.selector | indent 6 }}
-{{- end }}
```

Comment:
These line are still required for `nodeSelector`

Reply:
ok. got them back